i/influxdb/build_info.json

```diff
@@ -2,6 +2,8 @@
     "package_name" : "influxdb",
     "github_url":"https://github.com/influxdata/influxdb.git",
     "version": "v2.5.1",
+    "default_branch": "master",
+    "build_script" : "influxdb_ubi_8.5.sh",
     "package_dir": "i/influxdb/Dockerfiles/",
```

Comment:
I know this is not part of your changes, but just wondering, shouldn't the package_dir here be the top level directory "i/influxdb" as is the case for other packages?

Reply:
Yes, `package_dir` should be a top-level directory. But looks like in current case, it used for docker command.